crates/plugins/src/manifest.rs

```diff
@@ -30,6 +30,26 @@ impl PluginManifest {
     pub fn name(&self) -> String {
         self.name.to_lowercase()
     }
+
+    // Maps manifest Architecture options to associated Rust ARCH strings
+    // https://doc.rust-lang.org/std/env/consts/constant.ARCH.html
+    pub(crate) fn rust_arch(arch: &Architecture) -> String {
```

Comment:
I would expect these to be methods on either `PluginPackage` or on `Architecture`/`Os` (in which case they could be named e.g. `rust_name`).